### PR TITLE
[i18n] Update for Occitan

### PR DESCRIPTION
**Pull request by Mejans**
This is a first update. Will translate the admin pages later

### CODE REVIEW

File: locales/oc/date.po [hunk 1]
```diff
@@ -92,31 +92,31 @@ msgid "December"
 msgstr "Decembre"
 
 msgid "_Mon"
-msgstr ""
+msgstr "_Diluns"
```
Comment (franck-paul) [inline locales/oc/date.po]: Should be 4 chars max only as in other already translated languages (see fr)

File: locales/oc/date.po [hunk 2]
```diff
@@ -92,31 +92,31 @@ msgid "December"
...
 
 msgid "_Tue"
-msgstr ""
+msgstr "_Dimars"
```
Comment (franck-paul) [inline locales/oc/date.po]: See above

File: locales/oc/date.po [hunk 3]
```diff
@@ -92,31 +92,31 @@ msgid "December"
...
 
 msgid "_Wed"
-msgstr ""
+msgstr "_Dimècres"
```
Comment (franck-paul) [inline locales/oc/date.po]: See above

File: locales/oc/date.po [hunk 4]
```diff
@@ -92,31 +92,31 @@ msgid "December"
...
 
 msgid "_Thu"
-msgstr ""
+msgstr "_Dijòus"
```
Comment (franck-paul) [inline locales/oc/date.po]: See above

File: locales/oc/date.po [hunk 5]
```diff
@@ -92,31 +92,31 @@ msgid "December"
...
 
 msgid "_Fri"
-msgstr ""
+msgstr "_Divendres"
```
Comment (franck-paul) [inline locales/oc/date.po]: See above

File: locales/oc/date.po [hunk 6]
```diff
@@ -92,31 +92,31 @@ msgid "December"
...
 
 msgid "_Sat"
-msgstr ""
+msgstr "_Dissabte"
```
Comment (franck-paul) [inline locales/oc/date.po]: See above

File: locales/oc/date.po [hunk 7]
```diff
@@ -92,31 +92,31 @@ msgid "December"
...
 
 msgid "_Sun"
-msgstr ""
+msgstr "_Dimenge"
```
Comment (franck-paul) [inline locales/oc/date.po]: See above

**Comment by franck-paul** [comment]
Nice job ;-)

**Comment by Mejans** [comment]
Changes done!

File: locales/oc/date.po [hunk 8]
```diff
@@ -92,31 +92,31 @@ msgid "December"
 msgstr "Decembre"
 
 msgid "_Mon"
-msgstr ""
+msgstr "_Dl"
```
Comment (franck-paul) [inline locales/oc/date.po]: Better but still not perfect: example from french → lun. for lundi

See also the en date.po → Mon for monday

Comment (Mejans) [reply locales/oc/date.po]: We never use days in short =/
I can put Dil, Dim, Dim, Dij, Div, Dis, Dim but it's not that pretty :crying_cat_face:

Comment (franck-paul) [reply locales/oc/date.po]: > We never use days in short =/

I need to check where and how they are used in Dotclear…

Comment (Mejans) [reply locales/oc/date.po]: Well it's not that bad in 3 letters

Comment (Mejans) [reply locales/oc/date.po]: Tell me, as long as it's merged I'm happy :)

Comment (franck-paul) [reply locales/oc/date.po]: > > We never use days in short =/
> 
> I need to check where and how they are used in Dotclear…

En fait c'est utilisé pour afficher une date en clair. Je suppose qu'avoir le nom du jour en entier ne sera pas gênant, plutôt qu'une abréviation.

Du coup le mieux est de mettre 3 ou 4 caractères ou plus, du moment que ça reste compréhensible.

PS : Ne pas mettre le "underscore" dans la traduction.

Comment (Mejans) [reply locales/oc/date.po]: Voilà, je pense que pour un premier rafraîchissement on est pas mal !

**Pull request merged**